GameJam/Game/GameRenderer.cs

```diff
@@ -14,7 +14,11 @@ public class GameRenderer : IDisposable
         private Font font = new Font(FontFamily.GenericMonospace, 5); 
         private SolidBrush colourBrush = new SolidBrush(Color.White);
 
-        public bool isRenderingDialogue = true;
+        public DialogueSystem dialogueSystem;
+        public string dialogue;
+
+        public bool isRenderingDialogue = false;
+        public bool nextDialogue = false;

```

Comment:
```suggestion
        public bool isRenderingDialogue;
        public bool nextDialogue;

```
Booleans zijn standaard vals :P